public/app/features/alerting/unified/api/alertmanagerApi.ts

```diff
@@ -115,7 +122,7 @@ export const alertmanagerApi = alertingApi.injectEndpoints({
       query: () => ({
         url: `/api/alertmanager/${getDatasourceAPIUid(
           GRAFANA_RULES_SOURCE_NAME
-        )}/config/history?limit=${LIMIT_TO_SUCCESSFULLY_APPLIED_AMS}`,
+        )}/config/history?limit=${ALERTMANAGER_CONFIGURATION_CONFIGURATION_HISTORY_LIMIT}`,
```

Comment:
While here, we could change to use `params` instead

```suggestion
        url: `/api/alertmanager/${getDatasourceAPIUid(GRAFANA_RULES_SOURCE_NAME)}/config/history`,
        params: {
          limit: ALERTMANAGER_CONFIGURATION_CONFIGURATION_HISTORY_LIMIT,
        },
```